pkg/asset/agent/manifests/clusterimageset.go

```diff
@@ -31,11 +33,45 @@ func (*ClusterImageSet) Name() string {
 // Dependencies returns all of the dependencies directly needed to generate
 // the asset.
 func (*ClusterImageSet) Dependencies() []asset.Asset {
-	return []asset.Asset{}
+	return []asset.Asset{
+		// &releaseimage.Image{},
+		// &installconfig.InstallConfig{},
+	}
 }
 
 // Generate generates the ClusterImageSet manifest.
 func (a *ClusterImageSet) Generate(dependencies asset.Parents) error {
+
```

Comment:
I'm guessing this will be turned on in the future when we are generating the ZTP manifests from install-config.yaml. When that is active the ClusterImageSet will be generated for the user if one is not provided.

Reply:
Yes